pkg/cache/tas_cache_test.go

```diff
@@ -805,51 +801,6 @@ func TestFindTopologyAssignment(t *testing.T) {
 				},
 			},
 		},
-		"don't double-count TAS pods when computing the capacity": {
```

Comment:
FYI this scenario is moved to the new suite as "workload gets scheduled as the usage of TAS pods and workloads is not double-counted", since it was tricky to account for the Workloads' usage in this setup (see comment below)